config/secrets.yml

```diff
@@ -12,11 +12,14 @@
 
 development:
   secret_key_base: 575d1ff900b8efcf99c89980371365dc659ac3b3b8d6ee034ff27147a312a39c994eb2c189f06cf875bbcc645c7916080b95173b8dfe2b7063a34637a4a46541
+  link_checker_api_secret_token: KWceiCFx6GjzvKQh28EiRiUE
 
 test:
   secret_key_base: 46dbb3ca836e2c15ed6e4cb4b1d30c9b3e99ec83d472be4973c43fa5d6eaea84e49baeb5167bb77c82dde54edadda7d1abfa0b73c4d2d1f547542e804156612d
+  link_checker_api_secret_token: KWceiCFx6GjzvKQh28EiRiUE
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV["SECRET_TOKEN"] %>
+  link_checker_api_secret_token: <%= ENV["LINK_CHECKER_API_SECRET_TOKEN"] %>
```

Comment:
Might be good to use `ENV.fetch` here so there would be a clear error if it wasn't set.

Reply:
Adding a `fetch` blows up the test suite 😞

Reply:
That's weird... :/ Shouldn't it be using the `test` values :S

Reply:
When the `yml` is parsed, I expect it tries to populate all the key/values first then takes then takes the specific env.

Reply:
Oh ugh, I see. I suppose that makes sense.